src/app/panels/sources-panel/sources-panel.component.html

```diff
@@ -1,4 +1,4 @@
-<evt-panel [secondary]="true">
+<evt-panel evtAnnotator [secondary]="true">
```

Comment:
I would put the `evtAnnotator` directive in the content div, so that the annotator works only on text and not on tools in the header/footer